doc/manual/src/command-ref/env-common.md

```diff
@@ -44,6 +44,12 @@ Most Nix commands interpret the following environment variables:
     The Nix search path can also be extended using the `-I` option to
     many Nix commands, which takes precedence over `NIX_PATH`.
 
+    If `NIX_PATH` is not set or empty, Nix will look up the following locations for a revision of `nixpkgs`:
```

Comment:
The fallback also does not seem to be used with empty `NIX_PATH`:

```ShellSession
$ nix-channel --add https://nixos.org/channels/nixos-unstable nixpkgs
$ nix-channel --update
$ unset NIX_PATH
$ nix-instantiate -E '<nixpkgs>'
error: expression does not evaluate to a derivation (or a set or list of those)
$ NIX_PATH= nix-instantiate -E '<nixpkgs>'
error: file 'nixpkgs' was not found in the Nix search path (add it using $NIX_PATH or -I)
```

Reply:
I'm not too familiar with Nix. But locally, where I have an empty NIX_PATH, this command builds the go package for me:

`nix-build --no-out-link -E 'with import <nixpkgs> { }; go'`

Reply:
Are you sure you have the variable set to an empty string rather than unset?

Reply:
I tried:

unset NIX_PATH && nix-build --no-out-link -E 'with import <nixpkgs> { }; go' 
env NIX_PATH="" && nix-build --no-out-link -E 'with import <nixpkgs> { }; go'

and both are building to `/nix/store/n449plwz10c5l53qhfnrywyns0m8g0qv-go-1.17.11`

Reply:
@zuzuleinen You have to do
```console
 env NIX_PATH="" nix-build --no-out-link -E 'with import <nixpkgs> { }; go'
 ```
 instead. Note there is no `&&`, as just running `env` prints the environment, but running `env <assignments> <command>` runs the command with the assignments on top of the regular environment.

Reply:
Indeed running  `env NIX_PATH="" nix-build --no-out-link -E 'with import <nixpkgs> { }; go' ` will give me that error


```
error: file 'nixpkgs' was not found in the Nix search path (add it using $NIX_PATH or -I)

       at «string»:1:13:

            1| with import <nixpkgs> { }; go
             |             ^
(use '--show-trace' to show detailed location information)
```